orcid-integration-test/src/test/manual-test/README.md

```diff
@@ -102,26 +102,44 @@
 
 56. On https://qa.orcid.org/account add [orcid id] as a trusted individual
 
+## Test Self Service
+
+57. Set up a new Self Service Org account using [this document](https://docs.google.com/document/d/1FlpmgtubCEnD21EpJLWu4AXh4DH0moC9pe6R_5clA78/edit) or alternatively use 0000-0003-0641-4661.
+
+58. Sign in to 0000-0003-0641-4661 and visit https://aq.orcid.org/0000-0003-0641-4661/self-service
+
+59. Update org info- make sure it feeds to Salesforce.
+
+60. Add new consortium org contact- make sure info feeds to SF.
+
+61. Add new member org- make sure request feeds to SF.
+
+62. Set new org as a member in SF- see that the status updates.
```

Comment:
Note that SaleForce data are cached in the registry for an hour. Therefore, it will take up to an hour for updates made in SalesForce to appear in the registry.

(Updates made in self-service should appear instantly in SalesForce though.)

Reply:
Thanks @wjrsimpson please see latest PR!